react/components/EXPERIMENTAL_TableTree/Tree/CellPrefix.tsx

```diff
@@ -1,45 +1,35 @@
 import React, { FC } from 'react'
 
-import CaretRight from '../../icon/CaretRight/index.js'
-import CaretDown from '../../icon/CaretDown/index.js'
-import ButtonWithIcon from '../../ButtonWithIcon/index.js'
+import Arrow, { ArrowProps } from './Arrow'
+import Checkbox, { CheckboxProps } from '../../EXPERIMENTAL_Table/Checkbox'
 
-const Arrow: FC<ArrowProps> = ({ active, onClick }) => {
-  const icon = active ? <CaretDown /> : <CaretRight />
-  return (
-    <ButtonWithIcon
-      size="small"
-      onClick={onClick}
-      icon={icon}
-      variation="tertiary"
-    />
-  )
-}
+const PREFIX_WIDTH = 64
```

Comment:
Would be great if this was somehow customizable, I think this could easily vary between clients

Reply:
Considering the table usage this will not vary since arrows and checkboxes have fixed widths.

Reply:
Also, allowing users to customize it could lead to the `TableTree` looking uneven throughout different admin pages.

Reply:
Maybe it is something to discuss with @oldufre. A great solution, IMO, is to create a range of different sizes, such as: 
```
INDENTATION: {
  SMALL: 32,
  MEDIUM: 48,
  LARGE: 64
}
```
What do you think about it? 🤔

Reply:
I think that would be great, I just mentioned because I thought the alignment of the inner arrows was akward, I think they should be aligned to the outer checkboxes